ftplugin/clap_input.vim

```diff
@@ -73,6 +73,8 @@ nnoremap <silent> <buffer> <CR>      :<c-u>call clap#handler#sink()<CR>
 nnoremap <silent> <buffer> <C-c>     :<c-u>call clap#handler#exit()<CR>
 nnoremap <silent> <buffer> <C-g>     :<c-u>call clap#handler#exit()<CR>
 
+nnoremap <silent> <buffer> <Esc> <Esc>:<c-u>call clap#handler#exit()<CR>
```

Comment:
Just `:<c-u>call clap#handler#exit()<CR>` should be enough, and please also update the doc.

Reply:
Will do, but a maybe a day before I find the time